hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacility.java

```diff
@@ -116,4 +135,53 @@ public void handleTxBody(@NonNull final HederaState state, @NonNull final Transa
                     e);
         }
     }
+
```

Comment:
Will be nice to have javadocs on all methods.

Reply:
Added.